constants.py

```diff
@@ -0,0 +1,5 @@
+"""Constants"""
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Are all these imports necessary you are just defining two constants? I could be wrong if there is some issue with __future__ requiring all files to have the same imports.